### PR TITLE
added CoreCLR to the list of Microsoft projects on github

### PR DESCRIPTION
**Pull request by andyhammar**

### CODE REVIEW

**Comment by BethMassi** [comment]
Thanks @andyhammar for the PR. We should get @richlander to look at this one. I was under the impression that the .NET Core 5 project included this (.NET Foundation projects can have multiple repos). If it's a separate .NET Foundation project we should also look at adding it to the list of projects in the project folder. Otherwise, we just need to update the NETCore5.md with the additional repos.

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ There are many projects under the stewardship of the .NET Foundation. The [full
 **Microsoft**
 
 - [.NET Core 5](http://github.com/dotnet/corefx)
```
Comment (terrajobst) [inline README.md]: We should probably rename this to '. NET Core FX'

File: README.md [hunk 2]
```diff
@@ -23,6 +23,7 @@ There are many projects under the stewardship of the .NET Foundation. The [full
...
+- [CoreCLR](https://github.com/dotnet/coreclr)
```
Comment (terrajobst) [inline README.md]: ...and name this '.NET Core CLR'.

**Comment by richlander** [comment]
@terrajobst - that sounds good to me.

**Comment by BethMassi** [comment]
Done

**Pull request closed without merging**